website/newsletters/templates/newsletters/email.html

```diff
@@ -125,11 +125,11 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
                                 <br>
                                 <span style="font-family: Arial, sans-serif; font-size: 15px; color: white; ">
                                     {% if item.newsletterevent.start_datetime|date:'d-M-Y H:i' == item.newsletterevent.end_datetime|date:'d-M-Y H:i' %}
-                                        {{ item.newsletterevent.start_datetime }}
+                                        {{ item.newsletterevent.start_datetime|date:'D d-M-Y H:i' }}
```

Comment:
I believe this time format is fundamentally different from what is currently used:
> April 16, 2020, 8:30 p.m.

not that that is bad, because it would show
> April 16, 2020, 8:30 p.m. - 23:30 
in the case of line 130

But I would prefer
> Monday, 16 April 2020, 20:00 - 23:00

Reply:
In the feature request it stated that shorted weekdays were required. I'll let you decide if you want full or short week days and i'll change it accordingly.

Reply:
plus of course the formating change

Reply:
Oh short weekdays are also fine by me. The short months that are suggested are also perfectly fine. Just that it is not the ambiguous 'd-m-Y' or 'Y-m-d'